src/panels/Scene/SceneGraphNode/SceneGraphNodeView.tsx

```diff
@@ -33,29 +30,20 @@ export function SceneGraphNodeView({ uri }: Props) {
     );
   }
 
-  // We know that all scene graph nodes have the same subowners. However, not all of them
-  // are guaranteed to exist, so each of these may be undefined
-  const renderable = getSgnRenderable(propertyOwner, propertyOwners);
-  const scale = getSgnTransform(propertyOwner, TransformType.Scale, propertyOwners);
-  const translation = getSgnTransform(
-    propertyOwner,
-    TransformType.Translation,
-    propertyOwners
-  );
-  const rotation = getSgnTransform(propertyOwner, TransformType.Rotation, propertyOwners);
-
-  // Group the transforms under one tab, in the following order. Only show the transforms
-  // that are actually present
-  const transforms = [scale, translation, rotation].filter((t) => t !== undefined);
```

Comment:
Note that the order here was intentional, but maybe alphabetical makes more sense

My impression is that users most commonly fiddle with the scale propertyowner, which is why it was displayed first

Reply:
I personally think it's worth it, since they're open per default :)

Another alternative would be to use the order in which the property owners are added, i.e. not sort them at all. This _should_ match the order in which we want them presented in the UI (at least it does for all other subproperty owners), and if it doesn't we can add it on the C++ side

Reply:
Lol tried it and the order is literally the reverse of what we want. We could display it in reverse hehe?
![image](https://github.com/user-attachments/assets/fc2acd47-acba-4d5f-91d4-400d98dc09ac)